docs/loadtest_config.md

```diff
@@ -12,7 +12,7 @@ The URL to direct the load. Should be the public facing URL of the target Matter
 
 *string*
 
-The URL to the WebSocket endpoint the users will connect to.  
+The URL to the WebSocket endpoint the users will connect to.
```

Comment:
The double spaces were actually meant to break the line. I guess it might have been an automatic editor thing on save :P

Reply:
Ah I see :)